apps/web/src/server/lib/logger.js

```diff
@@ -0,0 +1,35 @@
+import config from '@pins/web/environment/config.js';
+import path from 'node:path';
+import pino from 'pino';
+
+/** @type {import('pino').TransportTargetOptions[]} */
+const targets = [];
+
+if (config.isProd) {
+	targets.push({
+		target: 'pino/file',
+		level: 'trace',
```

Comment:
Shouldn't this come from the env vars?

Reply:
It's not really necessary, as at the top level of the pino instantiation, the log level is passed from the `config`, which means logs won't be captured for anything lower level than the env var. Putting `trace` in the transport is basically saying "we're deferring to the master log level".